src/romaine/parser/simple.py

```diff
@@ -0,0 +1,181 @@
+class SimpleParser(object):
+    """
+        Gherkin single line parser for Romaine core.
+    """
+    def is_white(self, line):
+        """
+            Takes a line and determines whether it is pure whitespace.
+
+            Keyword arguments:
+            line - Line to test.
+
+            Returns:
+            True if the line is purely whitespace. False otherwise.
+        """
+        line = line.strip()
```

Comment:
This is fine, but what's the difference in behaviour between this and https://docs.python.org/2/library/stdtypes.html#str.isspace ?